lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -186,10 +175,8 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
      * @param hash
      * @param interactionType
      */
-    protected validateAndExtractStateFromHash(hash: string, interactionType: InteractionType, requestCorrelationId?: string): string {
+    protected validateAndExtractStateFromHash(serverParams: ServerAuthorizationCodeResponse, interactionType: InteractionType, requestCorrelationId?: string): string {
         this.logger.verbose("validateAndExtractStateFromHash called", requestCorrelationId);
-        // Deserialize hash fragment response parameters.
-        const serverParams: ServerAuthorizationCodeResponse = UrlString.getDeserializedHash(hash);
```

Comment:
Curious why does deserialization of hash fragment response parameters need to be moved to SilentFrameClient.ts and removed here?

Reply:
Because we need to extract the `state` parameter to send it to the native layer